### ISSUE TITLE
Variable "log" in post.py (line 400) is not defined

### ISSUE THREAD

**Issue by t-sommer** [conda/conda-build#3929]
See Linux builds in https://github.com/conda-forge/fmpy-feedstock/pull/15/checks?check_run_id=567079488 for an example.

```
Removed build tracker: '/tmp/pip-req-tracker-yjvhs8iv'

Resource usage statistics from building fmpy:
   Process count: 3
   CPU time: Sys=0:00:00.1, User=0:00:00.4
   Memory: 48.8M
   Disk usage: 3.8K
   Time elapsed: 0:00:04.1

Packaging fmpy
INFO:conda_build.build:Packaging fmpy
INFO conda_build.build:build(2074): Packaging fmpy
INFO:conda_build.build:Packaging fmpy-0.2.18-py36h95af2a2_0
INFO conda_build.build:bundle_conda(1388): Packaging fmpy-0.2.18-py36h95af2a2_0
Packaging fmpy-0.2.18-py36h95af2a2_0
compiling .pyc files...
number of files: 160
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 474, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 465, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2864, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2151, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1505, in bundle_conda
    files = post_process_files(metadata, initial_files)
File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1364, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/post.py", line 1176, in post_build
    post_process_shared_lib(m, f, prefix_files, host_prefix)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/post.py", line 1124, in post_process_shared_lib
    mk_relative_osx(path, host_prefix, m.config.build_prefix, files=files, rpaths=rpaths)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/post.py", line 400, in mk_relative_osx
    log.warn("Found Mach-O file but patching is only supported on macOS, skipping: %s", path)
NameError: name 'log' is not defined
```

**Comment by mingwandroid** [comment]
Thanks for the report.

**Comment by jjhelmus** [comment]
fixed in #3925 which will be included in the next release of conda-build.
